docs/contributing/development_environment.md

```diff
@@ -53,9 +53,39 @@ to create a virtual machine connected to the isolated networks.
 Create the edpm-compute-0 virtual machine.
 ```
 cd install_yamls/devsetup
-make standalone
+make standalone EDPM_COMPUTE_CEPH_ENABLED=false
 ```
 
+Create the edpm-compute-1 virtual machine for an extra Nova compute cell. Note
+that the main cell `nova` (in terms of OSP) is going to become `nova_cell1`,
+and this extra `cell1` we're adding into the source OSP deployment, is going
+to become `nova_cell2`, after the data plane adoption finishes.
```

Comment:
so if you followed our downstream docs you should only have 1 of two topologies

either you have a central stack with out multiple cells
in which case you have 3 databases
nova_api -> nova_api
nova_cell0 -> nova_cell0
nova -> nova_cell1

or you had two stacks

in which case there are no compute in the central stack

and you had 1 additional stack wtich computes

in that case
nova_api -> nova_api
nova_cell0 -> nova_cell0
nova_cell1 -> nova_cell1

in other word we do not need to copy the nova db in that case at all as it should have been unused.

it was possible to configure a hybrid topocy where the central stack had compute and there were additional cell stack but that was not supported in our product.

in this topology  we have two options

nova_api -> nova_api
nova_cell0 -> nova_cell0
nova -> nova_cell1
nova_cell1 -> nova_cell2

or 

nova_api -> nova_api
nova_cell0 -> nova_cell0
nova_cell1 -> nova_cell1
nova -> nova_cell2

or more generally 


nova_api -> nova_api
nova_cell0 -> nova_cell0
nova_cell1 -> nova_cell1
...
nova -> nova_celln

Reply:
> where the central stack had compute

that is the only way to go for Q4, i.e. having a single standalone controller+compute in the cell1